utils/add-file/README.md

```diff
@@ -1,5 +1,8 @@
 This spec is an example of manipulating files with Batch Changes. It adds a license file along each README.md found in any repository.
 
-It showcases:
-- [steps.files](https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#steps-files)
-- [template variables](https://docs.sourcegraph.com/batch_changes/references/batch_spec_templating#template-variables)
+Two different methods are demonstrated to add the file content:
+
+1. `add-file.batch.yaml` embeds the file content in the batch spec itself using the [`steps.files`](https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#steps-files) element. This has been supported since Sourcegraph 3.22, and can be useful to dynamically generate content using [templating](https://docs.sourcegraph.com/batch_changes/references/batch_spec_templating).
```

Comment:
```suggestion
1. `add-file.batch.yaml` embeds the file content in the batch spec itself using the [`steps.files`](https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#steps-files) element. This has been supported since Sourcegraph 3.22 and can be useful to dynamically generate content using [templating](https://docs.sourcegraph.com/batch_changes/references/batch_spec_templating).
```

Reply:
(Because it's VERY subtle in the diff, I dropped the unnecessary comma 😂)

Reply:
Oh no. That comma was intentional. Dialect fight. 😆 

The reason I added it was to separate the version requirement (which is of interest by itself) from the discussion of templating (which may not be as relevant for some people). In my head, they're separate (but related) clauses.

Let me see if I can reword a touch more to be clearer.

Reply:
OK, try it out now — I moved the discussion of the pros and cons of each method out into a new Markdown section.

Reply:
I like that, a lot clear. Thanks 🙂